joomla/joomla-cms
# [4.0] Check target file extension of com_media rename action

### Summary of Changes
Added a check in the rename action of com_media to make sure that the new extension does not violate the list of allowed extensions.


### Testing Instructions
1. Open browser dev tools
2. Open the rename dialog in com_media and hit save without changing the name
3. Inspect that last request in the network tab of the dev tools, edit it and change the file extension of the new_path parameter in the request body to a disallowed value, i.e. php
4. Verify that the file was renamed
5. Apply patch
6. Redo 1-3
7. Verify that the file could not be renamed anymore


### Actual result BEFORE applying this Pull Request
No check of extension


### Expected result AFTER applying this Pull Request
Check of extension
